### PR TITLE
Add basic project structure

### PR DESCRIPTION
**Pull request by gamb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "my-uix-project",
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "scripts": {
+    "start": "shadow-cljs watch app"
+  },
```
Comment (roman01la) [inline package.json:10]: would be cool to have prod build cmd here as well

File: src/app.cljs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+(defonce root
+  (uix.dom/create-root (js/document.getElementById "root")))
+
+(defn render
+  []
+  (uix.dom/render-root ($ app) root))
+
+(defn ^:export init
+  []
+  (render))
```
Comment (roman01la) [inline src/app.cljs:21]: looks like the app won't render since `init` is not called anywhere?

**Review by roman01la — CHANGES_REQUESTED** [review]

File: src/app.cljs (hunk 2 above)
Comment (gamb) [reply src/app.cljs:21]: Ah in shadow we have:
```
:modules {:main {:init-fn app/init}}
```
So this should get called when you `yarn start`…?

Comment (roman01la) [reply src/app.cljs:21]: would it work in a prod build as well? 🤔

Comment (diegoatpitch) [reply src/app.cljs:21]: It should. https://shadow-cljs.github.io/docs/UsersGuide.html#_modules
> :init-fn | Fully qualified symbol pointing to a function that should be called when the module is loaded initially.

File: package.json (hunk 1 above)
Comment (gamb) [reply package.json:10]: @roman01la would these aliases do?
```json
"build": "shadow-cljs compile app",
"release": "shadow-cljs release app"
```

Comment (roman01la) [reply package.json:10]: yep!

Comment (roman01la) [reply package.json:10]: what's `"build": "shadow-cljs compile app"` doing? just builds the project once in dev?

I think we only need dev watch build and release prod build

Comment (gamb) [reply package.json:10]: not too familiar but i believe that's right:

Compile a build once and exit (dev build? - not sure):
> $ shadow-cljs compile app

Running a release build optimized for production use:
> $ shadow-cljs release app

Comment (roman01la) [reply package.json:10]: let's keep release and watch ones

Comment (gamb) [reply package.json:10]: https://github.com/pitch-io/uix-starter/pull/3/commits/320e9701c0b0c2d1169f8d7fe2ab6a16eb2b6e46 looks ok?

Comment (roman01la) [reply package.json:10]: perfect!

**Pull request merged**